index.html

```diff
@@ -5,8 +5,8 @@
   <meta charset="utf-8">
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <title>Calculator</title>
-  <link rel="stylesheet" type="text/css" href="styles/style.css">
   <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css">
+  <link rel="stylesheet" type="text/css" href="styles/style.css">

```

Comment:
I have moved our CSS to be included after the Bootstrap CSS so that our rules override Bootstrap rules of the same specificity. This means that the margin on the bottom of the remove buttons is taken into account properly without modifying the rule. It is always best practice to have your own CSS after a framework as you will often be writing custom CSS to override the framework.